modules/canbus/vehicle/transit/protocol/llc_motionfeedback2_21_test.cc

```diff
@@ -0,0 +1,49 @@
+/******************************************************************************
+ * Copyright 2019 The Apollo Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *****************************************************************************/
+
+#include "modules/canbus/vehicle/transit/protocol/llc_motionfeedback2_21.h"
+#include "gtest/gtest.h"
+
+#include "modules/drivers/canbus/common/byte.h"
+#include "modules/drivers/canbus/common/canbus_consts.h"
+
+namespace apollo {
+namespace canbus {
+namespace transit {
+using ::apollo::drivers::canbus::Byte;
+
+class llc_motionfeedback2_21Test : public ::testing ::Test {
+ public:
+  virtual void SetUp() {}
+
+ protected:
+  Llcmotionfeedback221 motionfdk2_21_;
+};
+
+TEST_F(llc_motionfeedback2_21Test, motion_fdk) {
+  const uint8_t kData[8] = {0x9A, 0xFC, 0x56, 0xF7, 0x12, 0x34, 0xFF, 0xFF};
+  int32_t length = 1;
+
+  EXPECT_DOUBLE_EQ(motionfdk2_21_.llc_fbk_vehiclespeed(kData, length), 133.3);
```

Comment:
@sjiang2018 Different signals shall have different length, for instance , the llc_fbk_vehiclespeed has a length of 16 according to https://github.com/ApolloAuto/apollo/blob/0c9966e41561b8fcd4413da4054f9f048a27665b/modules/canbus/vehicle/transit/protocol/llc_motionfeedback2_21.cc#L55